src/test/java/teammates/test/pageobjects/AdminHomePage.java

```diff
@@ -95,4 +96,11 @@ public String getMessageFromResultTable(int index) {
         return getCellValueFromDataTable(index, 5);
     }
 
+    public String getJoinLink(int index) {
+        WebElement tableElement = browser.driver.findElements(By.className("table")).get(0);
+        WebElement trElement = tableElement.findElements(By.tagName("tr")).get(index);
+        WebElement tdElement = trElement.findElements(By.tagName("td")).get(5);
+        return tdElement.findElement(By.tagName("a")).getAttribute("href");
```

Comment:
Consider implementing a more general method `getLinkValueFromDataTable` in `AppPage` (follow the example of `getCellValueFromDataTable`).